core/arch/arm/kernel/generic_boot.c

```diff
@@ -299,6 +297,10 @@ static void init_runtime(unsigned long pageable_part)
 	tee_pager_add_pages((vaddr_t)__pageable_start + init_size,
 			(pageable_size - init_size) / SMALL_PAGE_SIZE, true);
 
+	/* Pager can also use pages before unpaged memory if any */
```

Comment:
It could be worth commenting on when and why there may be pages before unpaged memory.